monitoring/remoteRun.ts

```diff
@@ -32,18 +44,29 @@ async function main() {
 		'ca-central-1',
 	];
 
-	const invokeSettledResults = await Promise.allSettled(
-		regionsToCheck.map((region) =>
-			invokeInRegion(region, 'cmp-monitoring-CODE'),
-		),
-	);
-
-	invokeSettledResults.map((result) => {
-		if (result.status == 'fulfilled') {
-			processResult(result.value);
-		} else {
-			console.log('Failed to get response: ', result.reason);
-		}
+	await prompt([
+		{
+			type: 'list',
+			name: 'stage',
+			message: 'Which environment would you like to test?',
+			choices: ['prod', 'code'],
+		},
+	]).then(async (userInput: RemoteRunCLIUserInput) => {
```

Comment:
I think we should be able to select the regions to check against locally here (defaulting to all of them).